examples/segment_gatherer_msg_hour.yaml_template

```diff
@@ -0,0 +1,42 @@
+# Filename patterns that are collected
+patterns:
+  msg:
+    pattern:
+      "H-000-{orig_platform_name:4s}__-{orig_platform_name:4s}________-{channel_name:_<9s}-{segment:_<9s}-{start_time:%Y%m%d%H%M}-__"
+    # Process images hourly
+    hour_pattern: "00:00 23:59 01:00"
```

Comment:
This is not very obvious what each value is, could we use instead something like:
```yaml
hour_pattern:
    start_time: "00:00"
    end_time: "23:59"
    time_delta: "01:00"
```

Reply:
Yes, integrated.
But if I will produce for example the IR all the day and Visible only during the day, where I can configure this?
The gatherer sent using a topic and where, I can separate the processing ?

Reply:
In trollflow2 for example. There you can define for each composite a min or max sunlight coverage for example.